qa/integration-bwc/src/test/java/org/elasticsearch/action/admin/indices/template/BWCTemplateTests.java

```diff
@@ -20,15 +20,24 @@
 package org.elasticsearch.action.admin.indices.template;
 
 import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.index.mapper.MapperExtrasPlugin;
+import org.elasticsearch.plugins.Plugin;
 import org.elasticsearch.test.ESSingleNodeTestCase;
 
+import java.util.Collection;
+
 import static org.elasticsearch.test.StreamsUtils.copyToBytesFromClasspath;
 
 /**
  * Rudimentary tests that the templates used by Logstash and Beats
  * prior to their 5.x releases work for newly created indices
  */
 public class BWCTemplateTests extends ESSingleNodeTestCase {
+    @Override
+    protected Collection<Class<? extends Plugin>> getPlugins() {
+        return pluginList(MapperExtrasPlugin.class);
```

Comment:
I wonder if it makes more sense to put this in the mapper extra module. Are you doing this because you know it'll need some other module?

Reply:
I don't know but this is a possibility indeed.

Reply:
A few of us have been talking lately about trying to consolidate the qa modules rather than the one for this, one for that style that we have now. Given that desire I think it'd be better to move this into the module. If we have to move it back to a qa module later we can find it a better home then. Hopefully we won't though.